proto/skaffold.proto

```diff
@@ -190,7 +190,7 @@ message ResourceStatusCheckEvent {
 // PortEvent Event describes each port forwarding event.
 message PortEvent {
     int32 localPort = 1; // local port for forwarded resource
-    int32 remotePort = 2; // remote port is the resource port that will be forwarded.
+    IntOrString remotePort = 2; // remote port is the resource port that will be forwarded.
```

Comment:
unfortunately, this will be a backward incompatible change.  Can we instead add a another field and mark this as deprecated.